src/stan/math/prim/mat/prob/multi_normal_rng.hpp

```diff
@@ -20,10 +20,10 @@ namespace stan {
   namespace prob {
     using Eigen::Dynamic;
 
-    template <class RNG>
+    template <class RNG, typename T1, typename T2>
```

Comment:
How so? How do you make a function accept MatrixBase arguments without using template parameters? I'm baffled.